flink-core/src/main/java/org/apache/flink/configuration/CheckpointingOptions.java

```diff
@@ -102,13 +102,17 @@
 
     /** Whether to enable state change log. */
     @Documentation.Section(value = Documentation.Sections.COMMON_STATE_BACKENDS)
-    @Documentation.ExcludeFromDocumentation("Hidden for now")
     public static final ConfigOption<Boolean> ENABLE_STATE_CHANGE_LOG =
             ConfigOptions.key("state.backend.changelog.enabled")
                     .booleanType()
                     .defaultValue(false)
                     .withDescription(
-                            "Whether to enable state backend to write state changes to StateChangelog.");
+                            "Whether to enable state backend to write state changes to StateChangelog. "
+                                    + "If this config is not set explicitly, it means no preference "
+                                    + "for enabling the change log, and the value in lower config "
+                                    + "level will take effect. The default value 'false' here means "
+                                    + "if no value set (job or cluster), the change log will not be"
```

Comment:
@Zakelly Hi, I reviewed the pull request and, in general LGTM.
I found a minor error in the doc string. There should be a space at the end of the line. Or the generated doc will be "the change log will not beenabled".

Reply:
Yes, that's right. Thanks a lot for pointing this out! I have fixed this at [080b0ab](https://github.com/apache/flink/pull/16153/commits/080b0ab09afc18ee5e24bd8b536380273cae0d4c)

Thanks again!